Pack18/Task04/Program.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Выяснить является ли число степенью двойки, используя битовые операции.
+
+Console.WriteLine("Введите число:");
+var input = int.Parse(Console.ReadLine());
+var counter = 0;
+var item = 1;
+
+if (input == 0)
+{
+    Console.WriteLine("ноль не является степенью двойки");
+}
+while (item < input)
```

Comment:
Good solution. But it can be solved without loop. The main idea is that if the number is a power of two, then it has only one "1" in binary, right?
So let's use it:
((number & (number - 1)) == 0)

Try to understand how it works